<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Space Invaders</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Configuration</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="colorOverlay">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Color overlay</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="bkgd_enable">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text">
          <string>Image background</string>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="Screen" name="screen" native="true">
        <property name="styleSheet">
         <string notr="true">border-style: linet;
border-width: 4px;
border-color: white;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="scaleButton">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>x2</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="title">
         <string>DIP Switches</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Bonus life at : </string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="bonusLife">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>1000</number>
           </property>
           <property name="maximum">
            <number>1500</number>
           </property>
           <property name="singleStep">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="lives">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="accelerated">
            <bool>false</bool>
           </property>
           <property name="correctionMode">
            <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <number>3</number>
           </property>
           <property name="maximum">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="coinInfo">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Display coin info ?</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Lives : </string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="audioBox">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="title">
         <string>Audio</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Sound volume</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Mute</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDial" name="volume">
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>80</number>
             </property>
             <property name="tracking">
              <bool>true</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
             <property name="invertedControls">
              <bool>false</bool>
             </property>
             <property name="wrapping">
              <bool>false</bool>
             </property>
             <property name="notchTarget">
              <double>5.000000000000000</double>
             </property>
             <property name="notchesVisible">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Full</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Screen</class>
   <extends>QWidget</extends>
   <header>screen.hpp</header>
   <container>1</container>
   <slots>
    <slot>setBackgroundImage(bool)</slot>
    <slot>setColorOverlay(bool)</slot>
    <slot>setScale(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>colorOverlay</sender>
   <signal>clicked(bool)</signal>
   <receiver>screen</receiver>
   <slot>setColorOverlay(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorOverlay</sender>
   <signal>clicked()</signal>
   <receiver>screen</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bkgd_enable</sender>
   <signal>clicked(bool)</signal>
   <receiver>screen</receiver>
   <slot>setBackgroundImage(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
